types/mercadopago/resources/payment.d.ts

```diff
@@ -23,7 +23,9 @@ export interface MercadoPagoPayment {
 
   get(id: number, configuration?: DefaultConfigurationOmitQs, callback?: CallbackFunction): Promise<PaymentGetResponse>;
 
-  capturePartial(payload: CapturePartialPayload, configuration?: DefaultConfigurationOmitQs, callback?: CallbackFunction): Promise<PaymentCreateResponse>;
+  capture(id: number, configuration?: DefaultConfigurationOmitQs, callback?: CallbackFunction): Promise<CreatePaymentPayload>;
```

Comment:
Although is not technically wrong, you should change the response to the same as `capturePartial`, so it should be `Promise<PaymentUpdateResponse>`